shell/platform/config.gni

```diff
@@ -4,5 +4,5 @@
 
 declare_args() {
   # Whether or not to include desktop embedding targets in the build.
-  enable_desktop_embeddings = true
+  enable_desktop_embeddings = !is_fuchsia
```

Comment:
Should we migrate the comments about why this is not safe for Fuchsia over to this file? I feel like there should at least be a TODO here explaining why this can't be enabled on Fuchsia ATM.